static/bidder-info/emx_digital.yaml

```diff
@@ -1,6 +1,6 @@
 endpoint: "https://hb.emxdgt.com"
 maintainer:
-  email: "adops@emxdigital.com"
+  email: "contactaperturemx@cadent.tv"
```

Comment:
Sent an email to this email address for verification. Please reply "received".

Reply:
Is the contact email address for emx_digital to be changed as well?

Reply:
We've received and replied to the email address.  Both addresses (`cadent_aperture_mx` and `emx_digital`) should be changed to this new @cadent.tv address.